homework/03LessonThree.js

```diff
@@ -110,10 +115,14 @@ var threeByThreeArray = [
 
 // Problem 11:
 // Replace the center element of threeByThreeArray, which is 5, with something else.
+threeByThreeArray[1][1] = 'Charmander'
 
 // Problem 12:
 // create a 2 by 2 (2 rows, 2 columns) nested array
-var twoByTwoArray;
+var twoByTwoArray = [
+  [1,1],
+  [2,2]
+];
 
```

Comment:
Lol, awesome memes, and thanks for the feedback!

On Mon, Aug 8, 2016 at 12:14 PM, Piero A. Pretto <notifications@github.com>
wrote:

> In homework/03LessonThree.js
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/61#discussion_r73914671>
> :
>
> >
> >  // Problem 12:
> >  // create a 2 by 2 (2 rows, 2 columns) nested array
> > -var twoByTwoArray;
> > +var twoByTwoArray = [
> > +  [1,1],
> > +  [2,2]
> > +];
> >
>
> Outside the syntax issues that prevent your mocha tests from running, all
> your logic is 100%. Excellent work.
>
> [image: wizard]
> <https://camo.githubusercontent.com/48cc698d5897a8a8657f24b00f0b63bc9ca152f5/687474703a2f2f692e6d656d656361707461696e2e636f6d2f67656e645f696d616765732f376d533379672e706e67>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/61/files/063df41317298fb21bcd456e991c1acc7dab330d..43195828ce30ee9caf1dbc324923e52d911a5c0e#r73914671>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APUilqEbedv8ykLZ9snB7DhsPl8hlu3Cks5qd2PdgaJpZM4JUtxx>
> .
>



-- 
*Gabe Amaya*
*Front-End Web Developer/Designer*
Texas State University
512.644.4270
*GabeAmaya2@gmail.com <GabeAmaya2@gmail.com>*